python/lsst/ts/ess/sensors/command_handler.py

```diff
@@ -164,7 +164,7 @@ async def connect_devices(self):
             ess_instrument = EssInstrument(
                 configured_device["name"],
                 sel_temperature,
-                self.callback,
+                self._process_sensor_telemetry,
                 self.log,
             )
```

Comment:
Please consider using named arguments, to make it clearer what's going on -- especially with the new callback function.

Reply:
Done